### PR TITLE
WIP: MSI: Add addons and operators aws ipi workflows

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

File: ci-operator/step-registry/addon/aws/ipi/addon-aws-ipi-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    env:
+      COMPUTE_NODE_REPLICAS: "6"
+    post:
+    - ref: addon-uninstall
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - ref: addon-install
+  documentation: |-
+    This workflow provision an OpenShift cluster with a default configuration on AWS.
```
Comment (omrirh) [inline ci-operator/step-registry/addon/aws/ipi/addon-aws-ipi-workflow.yaml:14]:
```suggestion
    This workflow is to provision an OpenShift cluster with default configurations on AWS. 
```

File: ci-operator/step-registry/addons-and-operators/aws/ipi/addons-and-operators-aws-ipi-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    post:
+    - ref: operator-uninstall
+    - ref: addon-uninstall
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - ref: addon-install
+    - ref: operator-install
+  documentation: |-
+    This workflow provision an OpenShift cluster with a default configuration on AWS.
```
Comment (omrirh) [inline ci-operator/step-registry/addons-and-operators/aws/ipi/addons-and-operators-aws-ipi-workflow.yaml:16]: ditto

File: ci-operator/step-registry/operator/aws/ipi/operator-aws-ipi-workflow.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    env:
+      COMPUTE_NODE_REPLICAS: "6"
+    post:
+    - ref: operator-uninstall
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - ref: operator-install
+  documentation: |-
+    This workflow provision an OpenShift cluster with a default configuration on AWS.
```
Comment (omrirh) [inline ci-operator/step-registry/operator/aws/ipi/operator-aws-ipi-workflow.yaml:14]: ditto

**Comment by oharan2** [comment]
Moved to #41007

**Pull request closed without merging**